libs/ledger-live-common/src/families/cardano/specs.ts

```diff
@@ -163,6 +164,67 @@ const cardano: AppSpec<Transaction> = {
         );
       },
     },
+    {
+      name: "delegate to pool",
+      maxRun: 1,
+      transaction: ({ account, bridge, maxSpendable }) => {
+        invariant(
+          maxSpendable.gte(minBalanceRequiredForDelegate),
+          "balance is too low"
+        );
+        const transaction = bridge.createTransaction(account);
+        return {
+          transaction,
+          updates: [
+            {
+              mode: "delegate",
+              poolId:
+                "7df262feae9201d1b2e32d4c825ca91b29fbafb2b8e556f6efb7f549",
+            },
+          ],
+        };
+      },
+    },
+    {
+      name: "redelegate to pool",
```

Comment:
As it is now, we can't be sure this is doing a redelegate, right?

Could you maybe:
1. get the list of current delegations in the input `account`
2. use an `invariant` to make sure there's a least one
3. use any of them as the target `poolId`

Don't know if this makes sense, wdyt?

Reply:
Yes you're right, I'll update accordingly. same thing also required in undelegate